jsonld-contexts/sedimark-park.jsonld

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
never put the name of a project in the name of a context file. A context represents a **domain model**, it will continue to live after a project ends.